### ISSUE TITLE
Sounds have a monotone feel to them.

### ISSUE THREAD

**Issue by NathanLithia** [henkelmax/sound-physics-remastered#88]
**Bug description**
Sounds have a monotone feel to them. I am unsure if this is intentional for processing reasons. but it is incredibly distracting when using this mod with the ambientsounds mod etc

**Steps to reproduce the issue**
1. Install soundphysics-forge-1.19.2-1.0.18
2. Use any sound that should not be affected like the menu or is blacklisted in config.
3. Sounds have a monotone feel to them.

**Expected behavior**
Hear the sound as if the sound was played in vanilla when blacklisted

**Versions**
- Minecraft version, 1.19.2
- Forge/Fabric version, 43.2.3
- Mod version soundphysics-forge-1.19.2-1.0.18

**Screenshots (Optional)**
[off.webm](https://user-images.githubusercontent.com/10991641/211176762-7fa23ed6-2424-4e59-ac22-5c9de2669ab5.webm)
[on.webm](https://user-images.githubusercontent.com/10991641/211176764-0354823e-710f-4ba7-95eb-6dbcd95c145a.webm)

**Comment by henkelmax** [comment]
I can't really hear a big difference here. Does this happen without any other mods and on the latest version of the mod?

**Comment by NathanLithia** [comment]
> I can't really hear a big difference here. Does this happen without any other mods and on the latest version of the mod?

Yes, it happens without any other mods & the latest forge version of the mod 1.19.2-1.0.18
it's a huge difference for me.

**Comment by Shad0wolf0** [comment]
I do hear it too, especially if I turn off **Audio Enhancements** on Windows. I compared breaking sand on a Vanilla client and a Modded client with this mod.
